chart/queue-worker/README.md

```diff
@@ -0,0 +1,14 @@
+# JetStream for OpenFaaS queue-worker
+
```

Comment:
We will need a table at the end of this file explaining the various values available for the chart.